src/Symfony/Component/Cache/Traits/FilesystemTrait.php

```diff
@@ -23,6 +23,23 @@
     use FilesystemCommonTrait;
 
     /**
+     * @return bool
+     */
+    public function prune()
+    {
+        $time = time();
+        $pruned = true;
+
+        foreach (new \RecursiveIteratorIterator(new \RecursiveDirectoryIterator($this->directory, \FilesystemIterator::SKIP_DOTS), \RecursiveIteratorIterator::LEAVES_ONLY, \RecursiveIteratorIterator::CATCH_GET_CHILD) as $file) {
+            if ($time >= $file->getMTime()) {
```

Comment:
I mentioned this in https://github.com/symfony/symfony/pull/23451#issue-241419054, none of the maintainers have mentioned `mtime` should not be used in favor of the in-file time. We'll see how @nicolas-grekas want to handle this.